pom.xml

```diff
@@ -70,6 +70,11 @@
             <artifactId>slf4j-api</artifactId>
             <version>${slf4j.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
+        </dependency>
```

Comment:
FYI @ljucam @Kusig @linovifian 

Just that you are aware what this means: This will make our houston and eagle transitively depend on guava, with no option to exclude the dependency because redisques would not work anymore then.

Maybe this could be a deal-breaker for PaISA. But as I'm not aware of how broad the impact will be, I'll mention you here.

Reply:
FYI the same code (RedisMonitor) already exists in gateleen. Since I don't wanted to add a dependency to gateleen, I duplicated the code in vertx-redisques

Reply:
We already have guava explicitly in Eagle, and in Houston we use the `MonitoringHandler` from gateleen that transitievly uses guava.

We'd only have to check that this new explicit guava version doesn't break anything in these two services.